HACKING.md

```diff
@@ -156,6 +156,10 @@ done with the following command:
 
     $ prove -l t/js.t
 
+We also have a set of tests of client-side functions that can be run (after
+compiling) by opening http://localhost:5000/static/scripts/tests/web.html
+on your local development server.
+
```

Comment:
FWIW, this is the way I’m running URL Cleanup tests since these have been ported to ES6 and thus can no longer be run directly in Node (without Chrome). But it can also be run from the command-line by relying on a headless instance of Chrome in the background, which is how it runs in CircleCI:

```suggestion
We also have a set of tests of client-side functions that can be run (after
compiling) by opening http://localhost:5000/static/scripts/tests/web.html
on your local development server.

Provided your system is running a (headless) chrome instance, these client-side
tests can also be run from the command-line, using:

    $ node t/web.js
```